src/core/types/addons.js

```diff
@@ -1,9 +1,11 @@
 /* @flow */
 import type { AddonTypeType } from 'core/constants';
 
+export const STATUS_PUBLIC: 'public' = 'public';
```

Comment:
I personally think the constant here should be moved to core/constants

Reply:
I second @rebmullin's comment

Reply:
@rebmullin is right, but I am not even sure we really need a constant outside the component.

Reply:
So do I need to move this to core/constants or can we use 'public' directly?

Reply:
You can use a constant but only in the component file.

Reply:
also, I think this could just be `export const STATUS_PUBLIC = 'public';`

Reply:
mine dumbness++ :man_facepalming:

Reply:
hey @championshuttler, sorry for the confusion but I think what @willdurand suggests is better... to put this constant in the component file itself. If this ends up being a constant we need over and over we can move it back the core/constants file